overview/features.rst

```diff
@@ -0,0 +1,103 @@
+============================
+Features Overview
+============================
+
+This page gives an overview of some of the features available in Mattermost Team Edition and Enterprise Edition.
+
+Team Edition Features
+----------------------------------------------
+
+1. **1-1 and group messaging**
+
+.. image:: ../images/messaging.png
+
+2. **File sharing**
+
+.. image:: ../images/file_sharing.png
+
+3. **Custom Emojis**
+
+.. image:: ../images/custom_emojis.png
+
+4. **Webhooks**
+
+.. image:: ../images/webhooks.png
+
+5. **Slash commands**
+
+.. image:: ../images/slash_commands.png
+
+6. **Plug-in framework**
+
+.. image:: ../images/plugin_framework.png
+
+7. **Mobile applications**
```

Comment:
May want to consider adjusting screenshot size 

![screenshot 2017-12-15 17 45 10](https://user-images.githubusercontent.com/9354841/34063568-d974fbd4-e1c0-11e7-9c38-b7826ed8f4f1.png)